docs/examples/AMI430_live_test.ipynb

```diff
@@ -0,0 +1,463 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "*Testing on mock hardware*\n",
+    "\n",
+    "We have performed a lot of stand alone tests (tests with mocked hardware) in qcodes/tests/test_ami430.py. In perticular, we have tested: \n",
+    "- If the driver remembers the internal setpoints if these are given in cartesian, spherical and cylindrical coordinates\n",
+    "- Check that we send to correct setpoint instructions to the individual instruments if inputs are cartesian, spherical or cylindrical coordinates\n",
+    "- Test that we can call the measured parameters (e.g. cartesian_measured) without exceptions occuring. \n",
+    "- Check that instruments which need to ramp down are given adjustment instructions first\n",
+    "- Check that field limit exceptions are raised properly\n",
+    "- Check that the driver remembers theta and phi coordinates even if the vector norm is zero. \n",
+    "- Test that a warning is issued when the maximum ramp rate is increased \n",
+    "- Test that an exception is raised when we try to set a ramp rate which is higher then the maximum allowed value.\n",
+    "\n",
+    "Furthermore, in qcodes/tests/test_field_vector.py we have tested if the cartesian to spherical/cylindircal coordinate transformations and vis-versa has been correctly implemented by asserting symmetry rules. \n",
+    "\n",
+    "*Testing on actual hardware*\n",
+    "\n",
+    "However, we also need to test with actual hardware, which is documented in this notebook. On individual instruments we need to run the following tests: \n",
+    "- Test if communcation is established with the individual instruments\n",
+    "- Test that we can correctly set current values of individual sources\n",
+    "- Test that we can correctly measure current values of individual sources \n",
+    "- Test that we can put the sources in paused mode \n",
+    "- Test that the ramp rates are properly set \n",
+    "\n",
+    "With the 3D driver, we will test the following: \n",
+    "- Test that the correct set points are reached if we give inputs in cartesian, spherical or cylindrical coordinates \n",
+    "- Test that we can set theta and phi to non-zero values which are remembered if r is ramped from zero to r > 0. \n",
+    "- Test that instruments are ramped up and down in the correct order, with ramp downs occuring before ramp ups. \n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sys \n",
+    "sys.path.append(r\"C:\\Users\\a-sochat\\development\\Qcodes_QUTech\")\n",
```

Comment:
This is needed because the notebook is not in the root folder of the qcodes module. If the qcodes module is not in a location where python can find it, we need to specifically add the module path to the system path. If the user running this notebook has qcodes installed in a standard location then this is not necessary. I will remove the line from the notebook